### ISSUE TITLE
Plain text passwords

### ISSUE THREAD

**Issue by myisaak** [GabLeRoux/github-actions-unity-playground#2]
https://github.com/GabLeRoux/github-actions-unity-playground/blob/cb9eee00abe0b9607632ca51dbb98b62e03a01ce/ci/docker_build.sh#L10

Wouldn't one agree that's its unsafe to include passwords as plain text. 

Proposed solution: use secrets.

**Comment by GabLeRoux** [comment]
yes indeed, this is more of a proof of concept than anything else and is not yet in a working state by the way (if I remember well). This mostly comes from following related project:  
https://gitlab.com/gableroux/unity3d-gitlab-ci-example

Which in this case are environment variables. I'm not too familiar with github actions yet.

**Comment by myisaak** [comment]
I wouldnt mind creating a pull request since I'm going to use this on my own Unity projects.

**Comment by L4ZZA** [comment]
This is how you use secrets on _pipeline.yml_ `${{ secrets.UNITY_PASSWORD }}` you obviously need to create a secret variable from the repo `settings > secrets`.
